docs/llm/quickstart.md

```diff
@@ -52,7 +52,7 @@ node: {
     }
   }
 }
-' > TinyLlama-1.1B-Chat-v1.0/graph.pbtxt
+' >> TinyLlama-1.1B-Chat-v1.0/graph.pbtxt
```

Comment:
why appending?

Reply:
still working on going through the CI
manually it worked before and works after that change, so doesn't make a difference

Reply:
with appending we can not run this command twice